client/src/components/Members/Community/Head.members.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Head.members.css';
+import {ReactComponent as Github} from "../img/github-brands.svg";
+import {ReactComponent as LinkedIn} from "../img/linkedin-brands.svg";
+import {ReactComponent as Email} from "../img/mail.svg";
+import planet from "../img/planet_2.svg";
+
+import {HeadMembersLists as headLists} from './HeadMembersList.js';
+
+function HeadMembers() {
+
+      return (<div className="head-div">
+            <h1 className="community-header-head">
+                  Community Members

```

Comment:
I think "Members" will be more relevant here. "Community Members" doesnt look good in mobile view anyways. Afterall, the main title should be according to the navbar.